### ISSUE TITLE
No swift container info has migrated to new DB

### ISSUE THREAD

**Issue by madisonkeene** [gvlproject/genomespace#48]
none of my swift containers have been migrated over to the new GS :(

**Comment by ykowsar** [comment]
This is wired. Can you check with amazon as well?

**Comment by madisonkeene** [comment]
Amazon is fine

**Comment by ykowsar** [comment]
Oki We have a problem :(

On Thu, Sep 15, 2016 at 11:21 AM, Madison Flannery <notifications@github.com

**Comment by madisonkeene** [comment]
Marking this as low priority until we know whats happening re: gs-prod

**Comment by madisonkeene** [comment]
No tool info has migrated over either...

**Comment by madisonkeene** [comment]
OK - I grabbed the tool info from the database you set up when you first launched GS 2.0. Not sure if they're all there but it'll do - I skimmed the simpleDB tool table and there was nothing important.

Also copied over all the data format converters, the gs-dev server will need a restart to apply the fix but I vote we pile up a few more bug fixes before we do that.

**Comment by madisonkeene** [comment]
tl;dr- only thing missing is the Swift container info, and these were never migrated over when you did the data migration with Marco the first time either. I'd imagine the S3, dropbox, etc storage spec's are baked in to the script. Will email you details @ykowsar :)

**Comment by ykowsar** [comment]
OK. I finally find a solution. The script is useless. I need to revert to a bloody old version which is fucking stupid.  So I ended up writing a script for sending everything through umongo. It requires JSON because they are stupid on CSV :{. I am too tired to go through each script one more time I will do it next week.

Just to remember what I had done.

I need to find the Swift containers.
change the itemName to _id
remove the AWS attributes (why they are there in the first fucking place)
and inser as jason 
If I type one more character i will break the keyboard

**Comment by madisonkeene** [comment]
No need to do it for all of them just the swift container one :) and maybe the tools one too just to add the extra ones I didn't transfer but tbh I dont think thats crucial.

**Comment by madisonkeene** [comment]
Just need to take a dump of the DB as backup. Closing.
